common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -141,6 +141,22 @@ namespace com.keyman.keyboards {
       return this.displayLayer + '-' + this.coreID;
     }
 
+    static sanitize(rawKey: LayoutKey) {
+      if(typeof rawKey.width == 'string') {
+        rawKey.width = parseInt(rawKey.width, 10); // '' => NaN.
+      }
+      rawKey.width = (isNaN(rawKey.width as number) ? 0 : rawKey.width) || ActiveKey.DEFAULT_KEY_WIDTH;
```

Comment:
I think `''` needs to be equivalent to `DEFAULT_KEY_WIDTH`, not `0`; [as found on keyboards repo](https://github.com/keymanapp/keyboards/blob/a8733640dd8531fb9d6f8c63394cff5f81681a85/experimental/n/naijatype/source/naijatype.keyman-touch-layout#L307-L310).

Reply:
Already done as-is, as elaborated upon in https://github.com/keymanapp/keyman/pull/7486#discussion_r1000054770.

Reply:
Ah yes, missed that comment sorry! Then out of curiosity I just tried `NaN || 100` and that `== 100`. So we probably don't even need the `isNaN()` check!

Reply:
`rawKey.width ||= ActiveKey.DEFAULT_KEY_WIDTH;`?

Reply:
> Ah yes, missed that comment sorry! Then out of curiosity I just tried `NaN || 100` and that `== 100`. So we probably don't even need the `isNaN()` check!

https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Turns out `NaN` is documented as a 'falsy' value, so this'll totally work.  SGTM; TIL.